<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>289</width>
    <height>504</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Config PyNTA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="config_tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="camera_config">
      <attribute name="title">
       <string>Camera</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Camera:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="camera_model">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string notr="true">Can't be changed while running</string>
         </property>
         <property name="statusTip">
          <string notr="true" extracomment="Can't be changed while running"/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Init:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="camera_init">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Parameters passed to the camera while initializing</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Model:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="camera_camera_model">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string>Camera model. Can't be changed while running</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Exposure Time:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="camera_exposure_time"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Binning X</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="camera_binning_x"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Binning Y</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="camera_binning_y"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>ROI X1</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="camera_roi_x1"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>ROI X2</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="camera_roi_x2"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ROI Y1</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="camera_roi_y1"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>ROI Y2</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="camera_roi_y2"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Background File</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="camera_background">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Option not yet implemented</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QComboBox" name="camera_background_method">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Option not yet implemented</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Background Method</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="saving_config">
      <attribute name="title">
       <string>Saving</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="saving_directory"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Directory:</string>
         </property>
         <property name="buddy">
          <cstring>saving_directory</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="saving_filename_photo"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Image Filename</string>
         </property>
         <property name="buddy">
          <cstring>saving_filename_photo</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="saving_filename_video"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Video Filename</string>
         </property>
         <property name="buddy">
          <cstring>saving_filename_video</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="saving_filename_tracks"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Locations Filename</string>
         </property>
         <property name="buddy">
          <cstring>saving_filename_tracks</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="saving_filename_log"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Log Filename</string>
         </property>
         <property name="buddy">
          <cstring>saving_filename_log</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="saving_max_memory">
         <property name="toolTip">
          <string>Maximum memory to allocate while saving data</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Max Memory (MB)</string>
         </property>
         <property name="buddy">
          <cstring>saving_max_memory</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="other_config">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <widget class="QLineEdit" name="other_user_name"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>User Name:</string>
         </property>
         <property name="buddy">
          <cstring>other_user_name</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="gui_refresh_time"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>GUI Refresh Time (ms)</string>
         </property>
         <property name="buddy">
          <cstring>gui_refresh_time</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Logging Level:</string>
         </property>
         <property name="buddy">
          <cstring>logging_level</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="logging_level">
         <item>
          <property name="text">
           <string>None</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/icon_/Icons/avocado_calibrate.svg</normaloff>:/icon_/Icons/avocado_calibrate.svg</iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Debug</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Information</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Error</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="apply_button">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
